pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -1,130 +1,168 @@
-  Any individual setting is a Sass variable. Each setting begins with the `$theme-` prefix, and we provide a list of all available settings [in the USWDS documentation]({{ site.baseurl }}/documentation/settings/). The following is an example of a simple settings file:
+    In USWDS, settings are Sass variables that can be configured via Sass' `@use...with()` rule. You can find full instructions for configuring USWDS settings, along with a list of all available settings, on the [Settings page]({{ site.baseurl }}/documentation/settings/).
 
-  {:.site-terminal}
-  ```scss
-  @use "uswds-core" with (
-    $theme-image-path: "../uswds/img",
-    $theme-show-compile-warnings: true,
-    $theme-banner-background-color: "ink",
-    $theme-banner-link-color: "primary-light",
-    $theme-banner-max-width: "none",
-  )
-  ```
+2. **Create the foundation**: Build all USWDS styles from these settings.
 
-- **Create the foundation**: Build all USWDS styles from these settings.
+    Once you have set up your project's custom settings, it is time to pull in the USWDS source code from the `uswds` package.
 
-  The USWDS source code is the core of the design system. It contains all the styles for USWDS components as well as the design language of Sass tokens and functions used to build those components. USWDS source code has its own Sass entry point, which lives in the `node_modules` directory when you install USWDS with npm.
+    The `uswds` package collects all the Sass in the Design System. It contains all the styles for USWDS components as well as the design language of Sass tokens and functions used to build those components.
 
-  This is called `uswds` (or, more accurately, `uswds/_index.scss`), and it’s found in the `/packages` directory of the USWDS npm package. When you install with npm, the complete path is typically `./node_modules/@uswds/uswds/packages/uswds/_index.scss`.
+    To load the `uswds` package, you must provide a path to its `index.scss` entry point. If you installed USWDS 3 with npm, the complete path to this file is: `./node_modules/@uswds/uswds/packages/uswds/_index.scss`.
 
-- **Build new work on top of that foundation**: Finally, add any custom project styles built from design system code.
+3. **Build new work on top of that foundation**: Finally, add any custom project styles built from design system code.
 
-After you import the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
+    After you've loaded the USWDS source code, you can build new styles with USWDS design tokens, functions, and mixins. For the purposes of this guide, we won’t get into custom code, but the important thing to understand is that any custom code should follow the settings and USWDS source code in your Sass entry point.
```

Comment:
What if they did want to get into "custom code," is there a resource for that that we could add?